modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -3386,6 +3390,30 @@ public IgniteConfiguration setAuthenticationEnabled(boolean authEnabled) {
         return this;
     }
 
+    /**
+     * Returns {@code true} if SSL certificates transmission is enabled for cluster. Otherwise returns {@code false}.
```

Comment:
Please add more details: it's unclear what kind of transmission is mentioned.

Reply:
Done.